network/src/noise/handshake.rs

```diff
@@ -105,20 +105,23 @@ impl HandshakeAuthMode {
         }
     }
 
+    pub fn maybe_mutual(
+        trusted_peers: Arc<RwLock<HashMap<PeerId, HashSet<x25519::PublicKey>>>>,
+    ) -> Self {
+        HandshakeAuthMode::MaybeMutual(trusted_peers)
+    }
```

Comment:
what's the point of this function if you can write directly `HandshakeAuthMode::MaybeMutual(trusted_peers)` instead of `HandshakeAuthMode::maybe_mutual(trusted_peers)`?